types/chrome-remote-interface/chrome-remote-interface-tests.ts

```diff
@@ -18,7 +18,8 @@ import CDP = require('chrome-remote-interface');
         await Network.enable({});
         await Page.enable();
         await Page.navigate({ url: 'https://github.com' });
-        // TODO await Page.loadEventFired();
+        const loadEvent = await client['Page.loadEventFired']; // instead of: await Page.loadEventFired();
```

Comment:
Unfortunately this should be:

```ts
const loadEvent = await client['Page']['loadEventFired'];
```

apparently as per https://github.com/cyrus-and/chrome-remote-interface/blob/v0.30.1/lib/api.js#L56 (I actually have code that breaks in a racy manner if it didn't wait for the load event). However, supporting `'Page.loadEventFired'` apparently is just a one-line fix in `chrome-remote-interface`. Should I do the PR?

Reply:
Good catch! It's too bad the upstream typings are out of sync with this thinking. I think I'd rather try to get the upstream typings to change, but maybe the ideal solution is to change both. Thoughts?

Reply:
I'll send a PR to `chrome-remote-interface` tomorrow perhaps and see what @cyrus-and thinks.

Reply:
Great! It looks like something similar was already done for those using events (`.on(...)`) and not Promises: https://github.com/cyrus-and/chrome-remote-interface/issues/443

Reply:
Ok PR sent https://github.com/cyrus-and/chrome-remote-interface/pull/460

Reply:
Version updated, thanks!

Reply:
One more thing:

```suggestion
        const loadEvent = await client['Page.loadEventFired'](); // instead of: await Page.loadEventFired();
```

Totally forgot the parentheses :sweat_smile: